### ISSUE TITLE
操作可能なプログラムまたはバッチ ファイルとして認識されない問題

### ISSUE THREAD

**Issue by maitakehub** [NON906/mascotgirl#2]
非常に勉強になるプログラムの公開に感謝します、初心者質問で恐縮ですがバッチファイルの実行時に以下のようなエラーが発生する場合どうすれば良いでしょうか？

C:\mascotgirl>run.bat
'ot.py' は、内部コマンドまたは外部コマンド、
操作可能なプログラムまたはバッチ ファイルとして認識されていません。
'--chatgpt_apikey' は、内部コマンドまたは外部コマンド、
操作可能なプログラムまたはバッチ ファイルとして認識されていません。

C:\mascotgirl>run.bat
'ot.py' は、内部コマンドまたは外部コマンド、
操作可能なプログラムまたはバッチ ファイルとして認識されていません。
'--chatgpt_apikey' は、内部コマンドまたは外部コマンド、
操作可能なプログラムまたはバッチ ファイルとして認識されていません。

C:\mascotgirl>run.bat
'ascot.py' は、内部コマンドまたは外部コマンド、
操作可能なプログラムまたはバッチ ファイルとして認識されていません。

**Comment by NON906** [comment]
念のためお聞きしますが、run.batのエンコードはShift JISになっていますか？

**Comment by maitakehub** [comment]
> 念のためお聞きしますが、run.batのエンコードはShift JISになっていますか？

はい、該当フォルダでターミナルのchcpのコマンドで932が返ってきているので、Shift JISのエンコードがされていると思います。

**Comment by NON906** [comment]
そのrun.batを送付することは可能ですか？

**Comment by maitakehub** [comment]
ありがとうございます、こちらが実際に実行しているrun.batです。
APIキーだけ伏せ字にしておりますがそれ以外はそのままです。
[run.zip](https://github.com/NON906/mascotgirl/files/12582424/run.zip)

**Comment by NON906** [comment]
確認しました。

改行コードがCRLFではなくLFになっていたので、それが原因だと思います。
各種エディタでCRLFで保存し直せば正しく動作するはずです。
